broker/src/main/java/io/camunda/zeebe/broker/Broker.java

```diff
@@ -275,7 +201,7 @@ private BrokerInfo createBrokerInfo(final BrokerCfg brokerCfg) {
   }
 
   private AutoCloseable addBrokerAdminService() {
-    final var adminService = new BrokerAdminServiceImpl(partitions);
+    final var adminService = new BrokerAdminServiceImpl(partitionManager.getPartitions());
```

Comment:
Not sure whether `PartitionManagerImpl` will implement the admin access interface. As you noted in one of your other comments the partition manager is doing too many things as it is. So rather, I think `PartitionManagerImpl` will create and return an object which implements that interface. 

I don't want to pass in the partition manager, neither as interface nor the implementation class, because eventually admin service will use a different interface altogether

Reply:
But we're in agreement that we won't pass the list of partitions directly :smile: 

Fair enough, let's postpone it till we have the admin interface.